tests/plugins/widget/manual/inlineeditable.md

```diff
@@ -0,0 +1,17 @@
+@bender-tags: widget, bug, 698, 4.21.0
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, sourcearea, basicstyles, widget
+
+1. Open browser console.
+1. Select `t` (the last letter) from `Test`.
+1. Click the "Bold" toolbar button.
+1. Switch to the source mode.
+
+	**Expected** Editor is switched to the source mode.
+
+	**Unexpected** There is an error in the console and editor is not switched to the source mode.
+1. Switch back to the editing mode.
+
+	**Expected** Editor is switched to the editing mode.
+
+	**Unexpected** There is an error in the console and editor is not switched to the editing mode.
```

Comment:
Based on our [code style](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests) this additional indentation is unnecessary, let's remove this.
```suggestion
**Expected** Editor is switched to the source mode.

**Unexpected** There is an error in the console and editor is not switched to the source mode.

1. Switch back to the editing mode.

**Expected** Editor is switched to the editing mode.

**Unexpected** There is an error in the console and editor is not switched to the editing mode.
```